packages/base/catalog-entry.gts

```diff
@@ -44,6 +44,7 @@ export class CatalogEntry extends Card {
       return this[realmInfo]?.name;
     },
   });
+  @field thumbnailURL = contains(StringCard, { computeVia: () => null }); // remove this if we want card type entries to have images
```

Comment:
Set this to null because in the designs it looked like type cards don't have thumbnails. If this changes, we can update